Sources/StreamChatUI/ChatMessageList/ChatMessageListTitleView_Tests.swift

```diff
@@ -0,0 +1,65 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+@testable import StreamChatUI
+import XCTest
+
+private typealias ChatMessageListTitleView = _ChatMessageListTitleView<NoExtraData>
```

Comment:
This `typealias` should be probably in  `ChatMessageListTitleView` and should be public.